lib/postcode_validation/use_case/address_list.rb

```diff
@@ -22,7 +22,7 @@ def execute(postcode:, country:)
 
       def formatted(addresses)
         addresses.map do |address|
-          { street_address: address.street_address, place: address.place }
+          { street_address: address.street_address }
```

Comment:
so are we only going to be returning a street_address for the /addresses one? Do you have an example of a street_address it returns? Was there a reason we no longer want to return the place as well?